packages/main/src/types/TabContainerBackgroundDesign.js

```diff
@@ -0,0 +1,40 @@
+import DataType from "@ui5/webcomponents-base/dist/types/DataType.js";
+
+/**
+ * @lends sap.ui.webcomponents.main.types.TabContainerBackgroundDesign.prototype
+ * @public
+ */
+const TabContainerBackgroundDesigns = {
+	/**
+	 * Solid color
```

Comment:
Use periods at the end to match the other types. See what `PageBackgroundDesign.js` has as documentation for its values.